src/main/java/de/retest/web/AttributesProvider.java

```diff
@@ -11,56 +11,56 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 
-public class AttributeProvider {
+public class AttributesProvider {
 
 	public static final String ATTRIBUTES_FILE_PROPERTY = "de.retest.recheck.web.attributesFile";
 	public static final String DEFAULT_ATTRIBUTES_FILE_PATH = "/attributes.yaml";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=9&issues=AWQTHhXVYwZU9vvrXIVA&open=AWQTHhXVYwZU9vvrXIVA)

[ISSUE_KEY]: AWQTHhXVYwZU9vvrXIVA